src/sentry/web/frontend/base.py

```diff
@@ -419,27 +418,6 @@ def convert_args(self, request: Request, organization_slug=None, *args, **kwargs
 
         return (args, kwargs)
 
-    def get_allowed_roles(self, request: Request, organization, member=None):
```

Comment:
this doesn't seem to be used anywhere before right?

Reply:
Correct.